src/main/java/lotto/domain/Bonus.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.constants.ExceptionMessage;
+import lotto.constants.LottoConstants;
+
+public class Bonus {
+
+    private final int bonus;
+
+    private Bonus(final int bonus) {
+        this.bonus = bonus;
+    }
+
+    public static Bonus from(final int bonus) {
+        validate(bonus);
+        return new Bonus(bonus);
+    }
+
+    private static void validate(final int bonus) {
+        if (!isBoundary(bonus)) {
+            throw new IllegalArgumentException(
+                    String.format(ExceptionMessage.BONUS_OUT_RANGE, LottoConstants.MIN_NUMBER.getConstants(),
+                            LottoConstants.MAX_NUMBER.getConstants())
+            );
+        }
+    }
+
+    private static boolean isBoundary(final int bonus) {
+        return LottoConstants.MIN_NUMBER.getConstants() <= bonus && bonus <= LottoConstants.MAX_NUMBER.getConstants();
+    }
+
+    public static boolean isSameNumber(final List<Integer> given, final Bonus bonus) {
+        return given.stream().anyMatch(n -> n == bonus.getBonus());
```

Comment:
```suggestion
    public static boolean isSameNumber(final List<Integer> given) {
        return given.stream().anyMatch(n -> n == bonus);
```
이렇게 하면 되지 않나요? 매개변수로 Bonus를 받아 구현하신 이유가 궁금합니다 !

Reply:
static으로 선언하지 않았다면 그렇게 쓸 수 있겠네요!! 
static을 지양하자 해놓고 또 static을 쓴점 반성합니다!!
감사해요